pyorbital/tests/test_orbital.py

```diff
@@ -407,3 +415,23 @@ def test_63(self):
                       line2="2 37849  98.7092 229.3263 0000715  98.5313 290.6262 14.19554485413345")
         orb.get_next_passes(parser.parse("2019-10-21 16:00:00"), 12, 123.29736, -13.93763, 0)
         warnings.filterwarnings('default')
+
+
+@pytest.mark.parametrize('dtime, expected',
+                         [(datetime(2024, 6, 25, 11, 0, 18),
+                           np.datetime64('2024-06-25T10:44:18.234375')),
+                          (datetime(2024, 6, 25, 11, 5, 0),
+                           np.datetime64('2024-06-25T10:44:18.234375')),
+                          (np.datetime64('2024-06-25T11:10:00.000000'),
+                           np.datetime64('2024-06-25T10:44:18.234375')),
```

Comment:
The expected value is always the same, so you could define that in the test method.

Reply:
Oh, yes, you are right, the truth is constant here. Fixed.